java/code/src/com/redhat/rhn/frontend/strings/jsp/StringResource_en_US.xml

```diff
@@ -21615,7 +21615,7 @@ The Tree Path, Base Channel, and Installer Generation should always match. This
         </context-group>
       </trans-unit>
       <trans-unit id="sdc.details.activation.summary" xml:space="preserve">
-        <source>With a system profile specific activation key, this system can be re-registered using the 'rhnreg_ks' command-line utility.  The system will be re-registered with the same id, history, groups, and channels (unless the system's base channel changes).</source>
+        <source>With a system profile specific activation key, this system can be be re-registered with the same id, history, groups, and channels (unless the system's base channel changes).</source>
```

Comment:
`..., this system can be be re-registered...` there is a typo, double `be` :wink:

Reply:
Man, I had one task :man_facepalming: 
Thank you!

Reply:
:rofl: it's because of your too high skill level in founding and automating patterns: **re-re**gistration was clearly supposed to be preceded by **be be** :rofl: